earthpy/plot.py

```diff
@@ -112,11 +168,8 @@ def plot_bands(
 
     Returns
     ----------
-    tuple
-        fig : figure object
-            The figure of the plotted band(s).
-        ax or axs : axes object(s)
-            The axes object(s) associated with the plot.
+    ax or axs : axes object(s)
+        The axes object(s) associated with the plot.
```

Comment:
Does it make sense to add some specificity here by saying what class the returned axes are? e.g., does this return an array of matplotlib.axes._subplots.AxesSubplot objects, or something else?

Reply:
totally!